src/ProgressOnderwijsUtils/Collections/Maybe.cs

```diff
@@ -49,23 +66,40 @@ public TOut Extract<TOut>(Func<TOk, TOut> ifOk, Func<TError, TOut> ifError)
         /// Converts an untyped error message into a specific type of failed Maybe.  This operator is a  workaround to make it easy to create an error message without redundant type info.
         /// </summary>
         [Pure]
-        public static implicit operator Maybe<TOk, TError>(Maybe_Error<TError> err) => new Maybe<TOk, TError>(err);
+        public static implicit operator Maybe<TOk, TError>(Maybe_Error<TError> err)
+            => new Maybe<TOk, TError>(err);
 
         /// <summary>
         /// Converts an untyped error message into a specific type of failed Maybe.  This operator is a  workaround to make it easy to create an error message without redundant type info.
         /// </summary>
         [Pure]
-        public static implicit operator Maybe<TOk, TError>(Maybe_Ok<TOk> err) => new Maybe<TOk, TError>(err);
+        public static implicit operator Maybe<TOk, TError>(Maybe_Ok<TOk> err)
+            => new Maybe<TOk, TError>(err);
+
+        public bool TryGet(out TOk okValueIfOk, out TError errorValueIfError)

```

Comment:
Added: less prescriptive way to extract values.

Reply:
Staat wel een beetje raar tussen twee operator overloads in.

Reply:
Dit is een soort alternatief voor de volgende operator overload was de logica.  Maar met de deconstruct denk ik eigenlijk dat ik beter de volgende overload gewoon weg kan doen.